app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -31,12 +31,39 @@ ShortFormDataService = (ListingDataService, ListingConstantsService, ListingPref
       'referral'
     ]
     primaryApplicant: [
-      'appMemberId', 'contactId',
-      'noPhone', 'noEmail', 'noAddress', 'hasAltMailingAddress',
-      'email', 'firstName', 'middleName', 'lastName', 'preferenceAddressMatch',
-      'phone', 'phoneType', 'alternatePhone', 'alternatePhoneType', 'ethnicity',
-      'gender', 'genderOther', 'primaryLanguage', 'otherLanguage', 'race', 'sexualOrientation', 'sexualOrientationOther',
-      'xCoordinate', 'yCoordinate', 'whichComponentOfLocatorWasUsed', 'candidateScore'
+      'alternatePhone',
```

Comment:
Since this diff is hard to understand, I sorted this list alphabetically, removed `'race'` and `'ethnicity'`, and added `'raceEthnicity'`, `'asianOther'`, `'blackOther'`, `'indigenousOther'`, `'latinoOther'`, `'menaOther'`, `'pacificIslanderOther'`, `'whiteOther'`